lib/iris/tests/unit/analysis/maths/test__output_dtype.py

```diff
@@ -47,7 +47,7 @@ def setUp(self):
                                 np.subtract,
                                 np.multiply,
                                 np.power,
-                                np.divide]
+                                np.floor_divide]
```

Comment:
I've changed this because in Python 2 `np.divide` is floor division, but in Python 3 it is true division. So in Python 3 it shouldn't be in this list, but in the one below.

I didn't want to do a Python version check to decide which list to put it in. It turns out that in Python 3, `np.divide` is the same object as `np.true_divide`, but I didn't want to rely on that either. So I've taken the easy way out and replaced it altogether.